CHANGELOG.rst

```diff
@@ -2,6 +2,12 @@ CHANGELOG
 =========
 
 
+1.11.10 (unreleased)
```

Comment:
missing today's date

Reply:
I'll add the date just before closing PR in order to avoid rework